rdrf/rdrf/models/proms/models.py

```diff
@@ -181,13 +181,12 @@ def _get_cde_specification(self):
             ui = "integer-normal"
             if widget_spec:
                 ui = "integer-slider"
+            params = {"type": "text"}
+            if self.cde.min_value and self.cde.max_value:
```

Comment:
distinguish  between is None and is zero

zero is allowed as min or max

Reply:
changes made